src/handlers/pull_requests_assignment_update.rs

```diff
@@ -0,0 +1,68 @@
+use crate::github::retrieve_pull_requests;
+use crate::jobs::Job;
+use anyhow::Context as _;
+use async_trait::async_trait;
+use tokio_postgres::Client as DbClient;
+
+pub struct PullRequestAssignmentUpdate;
+
+#[async_trait]
+impl Job for PullRequestAssignmentUpdate {
+    fn name(&self) -> &'static str {
+        "pull_request_assignment_update"
+    }
+
+    async fn run(&self, ctx: &super::Context, _metadata: &serde_json::Value) -> anyhow::Result<()> {
+        let db = ctx.db.get().await;
+        let gh = &ctx.github;
+
+        tracing::trace!("starting pull_request_assignment_update");
+
+        // delete everything before populating
+        init_table(&db).await?;
+
+        let rust_repo = gh.repository("rust-lang/rust").await?;
+        let prs = retrieve_pull_requests(&rust_repo, &gh).await?;
+
+        // populate the table
+        for (assignees, pr_num) in prs {
+            for assignee in assignees {
+                let assignee_id = assignee.id.expect("checked");
```

Comment:
Preexisting (but maybe you can make a separate PR for this?): id on User probably shouldn't be optional.

Reply:
yeah this is something that confuses me, don't know exactly why our https://github.com/rust-lang/triagebot/blob/c52016ab56eddb5ce2ac51a7bde6a47b8653719a/src/github.rs#L19-L23 has the Id optional :shrug: 

I agree to remove the `Option` is a separate PR.